_commands/basics/ln.md

```diff
@@ -0,0 +1,83 @@
+---
+---
+
+ln
+--
+
+`ln` is used to create a link (symbolic or hard) to a file on disk. Default creates hard links.
+
+~~~ bash
+$ ln file1.txt file2.txt
+$ ln -s file1.txt file2.txt
+~~~
+
+<!--more-->
+
+### What is Linking?
```

Comment:
I really like this section! Nicely done. Would love to see a __So what do I want?__ section that describes you more often than not want to use symbolic links. i.e. if you don't know what to use, pick symbolic.